server/data/data-api/src/main/java/org/apache/james/sieverepository/api/SieveRepositoryManagementMBean.java

```diff
@@ -36,4 +43,16 @@
 
     void removeQuota(String user) throws SieveRepositoryException;
 
+    void addActiveSieveScript(String user, String toFileName, String fromScriptFileName) throws Exception;
+
+    InputStream getActive(String user) throws ScriptNotFoundException, StorageException;
+
+    default String sanitizeScriptName(String fileName) {
```

Comment:
maybe you can write some tests about this method anyway ?

Reply:
Where would have test case go? Also, the method seems too trivial to warrant a test case.

Reply:
We try to get a good code coverage and I don't see tests for this.

The name *test* is misleading IMO. A test also is *a feature specification* and *document* the behaviour. So even if the code snipset is easy, and should not contain error, documenting it and it's use still make sens.